src/views/Report/index.tsx

```diff
@@ -698,6 +794,138 @@ function Report(props: ReportProps) {
         },
     );
 
+    const [
+        changeSource,
+        {
+            loading: changeSourceLoading,
+        },
+    ] = useMutation<
+        UpdateReportChangeInSourceMutation,
+        UpdateReportChangeInSourceMutationVariables
+    >(
+        UPDATE_REPORT_CHANGE_IN_SOURCE,
+        {
+            onCompleted: (response) => {
+                const { errors, result } = removeNull(response?.updateReport);
+
+                if (errors) {
+                    notifyGQLError(errors);
+                }
+                if (result) {
+                    notify({
+                        children: 'Source changed successfully!',
```

Comment:
```suggestion
                        children: 'Change in source set successfully!',
```

We need to change the message accordingly in other mutations